app-frontend/src/app/components/pages/project/analyses/analyses/index.html

```diff
@@ -1,2 +1,34 @@
-<div>analyses</div>
-<ui-view></ui-view>
+<div class="sidebar-actions-group">
+  <div style="flex: 1;"></div>
+  <select class="btn btn-small btn-transparent"
```

Comment:
These should probably be hidden until we implement the api support for it.
@designmatty I propose just sticking a text search field there like we currently have for the lab analysis browse view

Reply:
@Lknechtli I think it would make sense to just delete this dropdown, which is similar to what we have done for project layer list UI. Essentially, I agree that we should not show anything unimplemented.

Reply:
I'm fine with nothing there for now.